code/modules/mob/mob_movement.dm

```diff
@@ -542,3 +542,10 @@
 /// Can this mob move between z levels
 /mob/proc/canZMove(direction, turf/target)
 	return FALSE
+
+/mob/Move()
+	. = ..()
+	if(client)
+		for(var/sound/s in GLOB.allAttachedSounds)
+			s.recalcClient(client)
+			world << "recalculating for [src]"
```

Comment:
uh... this is bad right? like, this should not be here, this is a debug print

Reply:
you see the [WIP] in the title?